woocommerce/woocommerce-android
# Issue 5793 - [Order Creation] Rename add customer button label and customer details title in order creation flow

Rename add customer button label and customer details title in order creation flow

Closes: #5793


### Description

Updates the strings for the customer button and customer details title in order creation flow.

### Testing instructions


First make sure "Order Creation" beta feature is enabled:
1 - Tap "Menu" in the bottom navigation
2 - Tap the settings button
3 - Tap "Beta features" button
4 - Enable "Order Creation"

After "Order Creation" beta feature is enabled:
1 - Tap on "Orders" in the bottom navigation
2 - Tap the FloatingActionButton ("+" button) 
3 - Tap on "Create order" option
4 - Check the button below "Customer" section: the label should be "Add customer details"
5 - Tap on "Add customer details" button
6 - Check the title: the label should be "Customer Details"

### Images/gif
<p float="left">
<img width="350" alt="Screen Shot 2022-02-23 at 11 38 49 AM" src="https://user-images.githubusercontent.com/14964993/155343006-1982f956-056f-44c2-abc5-7c73a7e5b3f8.png">
<img width="350" alt="Screen Shot 2022-02-23 at 11 38 55 AM" src="https://user-images.githubusercontent.com/14964993/155343013-ac1397a2-e39d-4a4f-9346-0cb5c0592e37.png">
</p>
